webhub/models.py

```diff
@@ -50,241 +48,54 @@ class Pcuser(models.Model):
     def __unicode__(self):
         return self.user.username
 
+# Post table stores details about posts
 
-#Post table stores details about posts
 
 class Post(models.Model):
-    #The owner of the post
+    # The owner of the post
     owner = models.ForeignKey(Pcuser, null=False, related_name='owner')
-    #title
-    title_post = models.CharField(max_length=300)
-    #description
-    description_post = models.CharField(max_length=2000)
-    #link to important documents
+    title_post = models.CharField(max_length=100,
+                                  validators=[
```

Comment:
Yes, we will do that. @ajay-sai please make sure that you are sanitizing input for the PeaceTrack models. I have already started doing this for the Malaria models since they weren't included last year.